o3de/o3de-multiplayersample
# Popcorn Fx Library Upgrade (2.17->2.18)

Upgrading popcorn fx library from 2.17->2.18

Test:
1) Upgrade [PopcornFx Plugin
](https://github.com/o3de/o3de-multiplayersample-assets/pull/166)
2) Open PopcornFxTest and enable fx
3) Notice fx don't play/work
4) Export library in PkFx Editor
5) Reopen editor and notice new assets play
